README.rst

```diff
@@ -23,175 +23,26 @@ Briefcase
 .. image:: https://badges.gitter.im/pybee/general.svg
     :target: https://gitter.im/pybee/general
 
-Tools to support converting a Python project into a standalone native
-application.
+Briefcase is a tool for converting a Python project into a standalone native application. You can package projects for:
```

Comment:
We haven't spelled this out anywhere, sadly, but we try to keep line length to 80 characters.